tools/src/main/java/org/apache/kafka/tools/FeatureCommand.java

```diff
@@ -0,0 +1,331 @@
+        }
+    }
+
+    private static void addDescribeParser(Subparsers subparsers) {
+        subparsers.addParser("describe")
+                .help("Describes the current active feature flags.");
+    }
+
+    private static void addUpgradeParser(Subparsers subparsers) {
+        Subparser upgradeParser = subparsers.addParser("upgrade")
+                .help("Upgrade one or more feature flags.");
+        upgradeParser.addArgument("--metadata")
+                .help("The level to which we should upgrade the metadata. For example, 3.3-IV3.")
+                .action(store());
+        upgradeParser.addArgument("--feature")
+                .help("A feature upgrade we should perform, in feature=level format. For example: `metadata.version=5`.")
+                .action(append());
+        upgradeParser.addArgument("--dry-run")
+                .help("Validate this upgrade, but do not perform it.")
+                .action(storeTrue());
+
+    }
+
+    private static void addDowngradeParser(Subparsers subparsers) {
+        Subparser downgradeParser = subparsers.addParser("downgrade")
+                .help("Upgrade one or more feature flags.");
+        downgradeParser.addArgument("--metadata")
+                .help("The level to which we should downgrade the metadata. For example, 3.3-IV0.")
+                .action(store());
+        downgradeParser.addArgument("--feature")
+                .help("A feature downgrade we should perform, in feature=level format. For example: `metadata.version=5`.")
+                .action(append());
+        downgradeParser.addArgument("--unsafe")
+                .help("Perform this downgrade even if it may irreversibly destroy metadata.")
+                .action(storeTrue());
+        downgradeParser.addArgument("--dry-run")
+                .help("Validate this downgrade, but do not perform it.")
+                .action(storeTrue());
+    }
+
+    private static void addDisableParser(Subparsers subparsers) {
+        Subparser downgradeParser = subparsers.addParser("disable")
+                .help("Disable one or more feature flags. This is the same as downgrading the version to zero.");
+        downgradeParser.addArgument("--feature")
+                .help("A feature flag to disable.")
+                .action(append());
+        downgradeParser.addArgument("--unsafe")
+                .help("Disable this feature flag even if it may irreversibly destroy metadata.")
+                .action(storeTrue());
+        downgradeParser.addArgument("--dry-run")
+                .help("Perform a dry-run of this disable operation.")
+                .action(storeTrue());
+    }
+
+    static String levelToString(String feature, short level) {
+        if (feature.equals(MetadataVersion.FEATURE_NAME)) {
+            try {
+                return MetadataVersion.fromFeatureLevel(level).version();
+            } catch (Throwable e) {
+                throw new RuntimeException("UNKNOWN " + level, e);
```

Comment:
Should we append throwable `e` in this exception?

Reply:
I think it is appended to the exception?

Reply:
Last time, I just saw the scala code, it didn't include the original error message. But now, I checked again, it doesn't even throw an exception. So we should update it.
```
if (feature.equals(MetadataVersion.FEATURE_NAME))
      try MetadataVersion.fromFeatureLevel(level).version
      catch { case _: Throwable => s"UNKNOWN [$level]"}
```